test api/REST Client/OData v4.http

```diff
@@ -0,0 +1,563 @@
+@clientid = 7e930f8e-fac4-43a6-a242-3a4079ac4553
+@clientsecret = clientSecret
+@scope = https://api.businesscentral.dynamics.com/.default
+@tenant = ffb102fa-38b1-4d6b-ad59-1192df615c1e
+@environmentName = Sandbox1
+@companyId = 3ac0aadc-5ae6-ee11-a201-6045bde98aeb
```

Comment:
We can get companyId from /companies endpoint. Check this PR : https://github.com/ciellosinc/bc-template/pull/3/files where you can find examples